src/main/java/frc/robot/subsystems/drive/SwerveModuleIOTalonFXSim.java

```diff
@@ -38,9 +38,12 @@ public class SwerveModuleIOTalonFXSim implements SwerveModuleIO {
     private final TalonFXConfiguration driveTalonFXConfiguration = new TalonFXConfiguration();
     private final TalonFXConfiguration turnTalonFXConfiguration = new TalonFXConfiguration();
 
-    private final VelocityVoltage velocityVoltage;
+    private final VelocityTorqueCurrentFOC velocityTorqueCurrentFOC;
+
```

Comment:
Is there a reason why these fields are so spaced out from each other with empty newlines?

Reply:
Fixed.